### PR TITLE
Changelog for v1.10.6

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.10.6](https://github.com/go-gitea/gitea/releases/tag/v1.10.6) - 2020-03-10
+
+This is a retag version of v1.10.5 and also pin xgo version to go1.13.
```
Comment (jolheiser) [inline CHANGELOG.md]:
```suggestion
This is a re-tag version of v1.10.5 and also explicitly built with Go 1.13.
```

**Review by guillep2k — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ This changelog goes through all the changes that have been made in each release
...
+
+WARNNING: 1.10.5 is 100% busted (in the sense that it's actually 1.12).
```
Comment (jolheiser) [inline CHANGELOG.md]:
```suggestion
WARNING: v1.10.5 is incorrectly tagged targeting 1.12-dev and should **not** be used.
```

**Review by jolheiser — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (guillep2k) [inline CHANGELOG.md]:
```suggestion
WARNING: 1.10.5 is 100% busted (in the sense that it's actually 1.12).
```

**Review by mrsdizzie — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (lunny) [reply CHANGELOG.md]: @guillep2k It seems @jolheiser 's suggestion is better. Just followed his. :)

**Review by jolheiser — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

**Comment by zeripath** [comment]
Please send forward ports

**Pull request merged**